Explorer/Assets/DCL/Multiplayer/Connections/Credentials/ConnectionStringCredentials.cs

```diff
@@ -0,0 +1,31 @@
+using DCL.Utilities.Extensions;
+using System;
+using System.Web;
+
+namespace DCL.Multiplayer.Connections.Credentials
+{
+    public readonly struct ConnectionStringCredentials : ICredentials
+    {
+        private readonly string connectionString;
+
+        public ConnectionStringCredentials(string connectionString)
+        {
+            this.connectionString = connectionString;
+        }
+
+        public string Url => connectionString
+                            .Split('?')
+                            .EnsureNotNull("Nothing to split via: {}")[0]!
+                            .Replace("livekit:", string.Empty);
```

Comment:
you can cache it once in `ctor` to avoid string operations on each `Url` get-call (in case it is used more than once)

Reply:
I prefer to leave ctors code free, they only builds the objects, if cache is required it could be covered with cache decorator, see example settingsContainer.GetSettings<FontsSettings>().AsCached()